### ISSUE TITLE
Idea: propagate output label/doc to the output object

### ISSUE THREAD

**Issue by mr-c** [common-workflow-language/common-workflow-language#566]

**Comment by mr-c** [comment]
@tetron If I implement this, do you mind if we add it back to v1.1?

**Comment by tetron** [comment]
Sure.  There was a question about how to express metadata for non-object output parameters (numbers/strings/etc).

**Comment by mr-c** [comment]
Right; I'll do this for object output parameters for v1.1 and we can consider again this for the v2/v3 series
